.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+/submissions/kissik/port-sniffer/node_modules
```

Comment:
You can have a `.gitignore` in your task subdirectory thus keeping project `.gitignore` intact.
This is good for scoping and also would be helpful if you decide to make your task an independent project for your portfolio.

Reply:
Thank you for code-review.
I think I made all the necessary changes